rfcs/react-components/components/simple-component-implementation.md

```diff
@@ -0,0 +1,124 @@
+# RFC: Component recipes
+
+---
+
+@sopranopillow
+
+## Summary
+
+Decide how to implement simple components that could be built with already created components.
+
+## Background
+
+Currently Persona's spec calls for a media slot that allows the use of an Avatar, image, icon, and a PresenceBadge. When implementing Persona we considered what Persona provides, it is a grid that aligns its text and media based on the given label position. This is also known as a media object by other libraries.
+
+There's one key aspect that Persona adds that makes creating the component compelling, unifying the look and feel across apps. To achieve this, we don't need the image and icon media options, we only need the Avatar and PresenceBadge (it also makes more sense that a persona only showcases person-related information). This then is the reason why the already implemented Persona only supports these slots. But there's still the question about how to add the look and feel of Fluent to media objects that have an image or icon.
+
+This is where this RFC will decide how to proceed with this portion of the problem.
+
+## Problem statement
+
+As we start working on new components, there might be some components that are simple and could be built with a simple layout and the already finished components. When creating these components, we currently don't have a guidance on whether to let the user build the component or us provide the component. This RFC offers three options, one that will require the user to build the components using instructions (recipes), another where we will provide the implementation of the component, and a combination of both option 1 and 2 providing a partial implementation and a recipe.
+
+#### What makes a component a candidate for this RFC's solution?
+
+- Its implementation is simple.
+- The desired design is achievable adjusting the props of the inner components.
+- It doesn't have complex accessibility requirements.
```

Comment:
Consider if providing accessibility in complex scenarios is actually a great value a layout component can provide.  We might need to think about patterns and infrastructure to help coordinate interaction across components composed into a layout.

Reply:
After thinking about it, I'll remove this requirement. I think maybe this should be left up to the engineer, there might be some scenarios in which providing some (maybe not super detailed) guidance is worthwhile.